src/admin-views/widgets/components/dropdown.php

```diff
@@ -11,34 +11,39 @@
  *
  * @version TBD
  *
- * @var string $label Label for the dropdown.
- * @var string $value Value for the dropdown.
- * @var string $id    ID of the dropdown.
- * @var string $name  Name attribute for the dropdown.
+ * @var string $label      Label for the dropdown.
+ * @var string $value      Value for the dropdown.
+ * @var string $id         ID of the dropdown.
+ * @var string $name       Name attribute for the dropdown.
+ * @var string $dependency The dependency attributes for the control wrapper.
  */
 
 ?>
-<p
-		class="tribe-widget-dropdown tribe-common-form-control-dropdown"
+<div
+		class="tribe-widget-form-control tribe-widget-form-control--dropdown"
+		<?php
+		// Not escaped - contains html ( data-attr="value")
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: b309aa2e0d3217abd19ae1cfa22c855c</sub>